src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyCodeFixProvider.cs

```diff
@@ -120,10 +120,20 @@ private async Task<Solution> ProcessResultAsync(CodeFixContext context, Diagnost
 
             // Now, rename all usages of the field to point at the property.  Except don't actually 
             // rename the field itself.  We want to be able to find it again post rename.
-            var updatedSolution = await Renamer.RenameAsync(fieldLocations, propertySymbol.Name,
-                location => !location.SourceSpan.IntersectsWith(declaratorLocation.SourceSpan) &&
-                            CanEditDocument(solution, location.SourceTree, linkedFiles, canEdit),
-                symbols => HasConflict(symbols, propertySymbol, compilation, cancellationToken),
+            //
+            // We're asking the rename API to update a bunch of references to an existing field to the same name as an
+            // existing property.  Rename will often flag this situation as an unresolvable conflict because the new
+            // name won't bind to the field anymore.
+            //
+            // To address this, we let rename know that there is no conflict if the new symbol it resolves to is the
+            // same as the property we're trying to get the references pointing to.
+
+            var updatedSolution = await Renamer.RenameAsync(
+                fieldLocations.Filter(
+                    location => !location.IntersectsWith(declaratorLocation) &&
+                                CanEditDocument(solution, location.SourceTree, linkedFiles, canEdit)),
+                propertySymbol.Name,
+                nonConflictSymbols: ImmutableHashSet.Create<ISymbol>(propertySymbol),
                 cancellationToken).ConfigureAwait(false);
```

Comment:
two changes here.  first, we have already gotten the rename locations above, so here we're only filtering it.  second, instead of passing a callback function to not error on symbols that bind to the property post-rename, we just pass the property into the api, and the api will do that check.